packages/sample-apps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -14,7 +14,7 @@
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
     <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
-    <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
+    <AppxPackageSigningEnabled>False</AppxPackageSigningEnabled>
```

Comment:
Why this change?
Same with three other vcxproj change

Reply:
Usually removal of the keys causes the UWP apps not run. It would be nice not to do the signing related changes.

Reply:
That is what's breaking the CI:
```
Error: The package or bundle is not digitally signed or its signature is corrupted.
× Installing new version of the app
- Failed to deploy: Installing new version of the app
```

Reply:
Yes, I can revert it. This started with keys being removed bc they weren't exempted from the gitignore, but we also have some apps with signing disabled, and was curious if that would break anything.

Wondering if we've only ever tested the other apps with deploy from layout, or why they wouldn't see issues?